test/compress/pure_funcs.js

```diff
@@ -772,13 +772,6 @@ inline_pure_call_3: {
         console.log(a);
     }
     expect: {
-        var a = function() {
-            console.log("PASS");
-        }();
-        console.log(a);
+        console.log(void 0);
     }
-    expect_stdout: [
-        "PASS",
-        "undefined",
-    ]
```

Comment:
I think expect_stdout should be reinstated for this test since the result is invariant to `compress` options.